src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Interaction/SelfInvocation/SelfInvocation.vue

```diff
@@ -7,7 +7,7 @@
     ref="messageRef"
   >
     <label
-      class="name group px-px hover:text-skin-message-hover hover:bg-skin-message-hover relative min-h-[1em]"
+      class="name group px-px hover:text-skin-message-hover hover:bg-skin-message-hover relative min-h-[1em] w-full"
```

Comment:
This is causing an undesired effect see below video. This does not seem to be the cause if the issue. There is real issue that needs to be fixed. The corresponding interaction should not have width at the first place.

https://github.com/mermaid-js/zenuml-core/assets/169417/1b866957-c33d-4fd0-aaf8-0cbd3e21d0c2